ci-operator/step-registry/hypershift/azure/create/hypershift-azure-create-chain.yaml

```diff
@@ -36,14 +41,20 @@ chain:
         PULL_SECRET_PATH="/tmp/.dockerconfigjson"
       fi
       
+      ENCRYPTION_KEY_ID=""
+      if [[ $HYPERSHIFT_ETCD_ENCRYPTION == "true" ]]; then
+        ENCRYPTION_KEY_ID="$(cat "${SHARED_DIR}/hypershift_azure_etcd_encryption_active_key_url")"
+      fi
+      
       /usr/bin/hypershift create cluster azure ${ICSP_COMMAND} \
         --name ${CLUSTER_NAME} \
         --node-pool-replicas ${HYPERSHIFT_NODE_COUNT} \
         --base-domain ${HYPERSHIFT_BASE_DOMAIN} \
         --pull-secret ${PULL_SECRET_PATH} \
         --azure-creds ${CLUSTER_PROFILE_DIR}/osServicePrincipal.json \
         --location ${HYPERSHIFT_AZURE_LOCATION} \
-        --release-image ${RELEASE_IMAGE}
+        --release-image ${RELEASE_IMAGE} \
+        --encryption-key-id=${ENCRYPTION_KEY_ID}
```

Comment:
does this flag behave properly with an empty value?

Reply:
cc @bryan-cox

Reply:
Mulham worked this particular story but it looks fine if its empty to me here - https://github.com/openshift/hypershift/blob/dbbdfe06865e6fdb510c03babe88a4ac3041febb/cmd/cluster/azure/create.go#L119

Reply:
Yup, I think so. Please see the following job:
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/52707/rehearse-52707-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-azure-ipi-ovn-hypershift-guest-f14-destructive/1798060397291376640